data/AlmaLinux-9.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
IMO we should ads $osfamily-$version to hiera.yaml

Reply:
I actually prefer that too. Standby...

Reply:
Actually, that gets messy because of `path: '%{facts.os.name}-%{facts.os.release.major}.yaml'`. I will add a subdir.

Reply:
Done. I added `data/osfamily` and `data/osfamily/<major>` to try to prevent confusion.